src/pages/home/index.tsx

```diff
@@ -0,0 +1,152 @@
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Filler,
+  Legend,
+  BarElement,
+  LineController,
+  BarController,
+  CoreScaleOptions,
+} from 'chart.js';
+import { Line, Bar, Chart } from 'react-chartjs-2';
+
+import mockdata from 'mocks/mockdata.json';
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Filler,
+  Legend,
+  LinearScale,
+  CategoryScale,
+  BarElement,
+  PointElement,
+  LineElement,
+  Legend,
+  Tooltip,
+  LineController,
+  BarController,
+);
+
+const Home = () => {
+  const Data = mockdata.response;
+  const arrayOfData = [
+    ...Object.keys(Data).map((el, i) => {
+      const values = Object.values(Data);
+
+      return {
+        key: i,
+        id: values[i].id,
+        value_area: values[i].value_area,
+        value_bar: values[i].value_bar,
+        time: el,
+      };
+    }),
+  ];
```

Comment:
여기서 `value_area`랑 `value_bar`만 사용하시는 거 같은데 key랑 time을 넣어서 새로 만드시는 이유가 있으신가요?
그리고 map 안에서 value를 사용하셔서 시간복잡도가 제곱되는 거 같아요. 45번 라인에 console.log(value) 찍어보니 로그가 많이 찍힙니다. 이렇게 하시는 건 어떤가요?
```suggestion
const arrayOfData = Object.entries(Data).map(([key, value], i) => ({
  key: i,
  id: value.id,
  value_area: value.value_area,
  value_bar: value.value_bar,
  time: key,
}));
```